src/components/Login.js

```diff
@@ -0,0 +1,44 @@
+import { useState } from "react";
+import { Link } from "react-router-dom";
+import { useNavigate } from 'react-router';
+import { loginUser } from "../utils/requestUtils/AuthRequestUtils";
+import { Button, Checkbox, FormControl, FormControlLabel, Input, InputLabel } from "@mui/material";
+
+export const Login = () => {
+    const [userData , setUserData] = useState({email: '',password: ''});
+    const navigate = useNavigate();
+    const loginHandler =  async (event) => {
+        event.preventDefault();
+        const response = await loginUser(userData);
+        if(response && response.encodedToken){
+            navigate('/');
+        } else{
+            console.error(response.errors[0]);
+        }
```

Comment:
You might need to save the token for authorization

Reply:
Token is getting saved during Sign up itself so didn't need to save it again during login. Also, couldn't test just the login feature without signup first as data is getting lost in mockbee after each refresh.